docs/spring-boot/app-starter.md

```diff
@@ -97,7 +97,10 @@ bdk-app:
         allowed-method: ["POST", "GET"] # list of HTTP methods to allow
         allowed-headers: "*" # list of headers that a request can list as allowed (multiple values allowed by using ["header-name-1", "header-name-2"])
         exposed-headers: ["header-name-1", "header-name-2"] # list of response headers that a response can have and can be exposed, the value "*" is not allowed for this field.
-
+    tracing:
```

Comment:
do we have a reason to make it optional?

Reply:
yes maybe if you have your own strategy strategy and don't want us to mess with the MDC